sentry_sdk/transport.py

```diff
@@ -225,6 +315,7 @@ def _send_event(
 
         if self._check_disabled("error"):
             self.on_dropped_event("self_rate_limits")
+            self.record_lost_event("ratelimit_backoff", data_category="error")
```

Comment:
```suggestion
            self.record_lost_event("rate_limit", data_category="error")
```

Can we use just `rate_limit` instead?